tests/func/test_init.py

```diff
@@ -1,8 +1,11 @@
+import logging
 import os
 
+from dvc.compat import fspath
+from dvc.config import Config
 from dvc.exceptions import InitError
 from dvc.main import main
-from dvc.repo import Repo as DvcRepo
+from dvc.repo import Repo as DvcRepo, Repo
```

Comment:
```suggestion
from dvc.repo import Repo as DvcRepo
```
? Kudos DS for catching this one :slightly_smiling_face:

Reply:
should have paid more attention to DS, sorry